torch/ao/quantization/quantization_mappings.py

```diff
@@ -76,13 +76,16 @@
     nnqat.Linear: nnq.Linear,
     nnqat.Conv2d: nnq.Conv2d,
     nnqat.Conv3d: nnq.Conv3d,
+    nnqat.EmbeddingBag: nnq.EmbeddingBag,
 }
 
 # Default map for swapping float module to qat modules
 DEFAULT_QAT_MODULE_MAPPINGS : Dict[Callable, Any] = {
     nn.Conv2d: nnqat.Conv2d,
     nn.Conv3d: nnqat.Conv3d,
     nn.Linear: nnqat.Linear,
+    nn.EmbeddingBag: nnqat.EmbeddingBag,
+    nn.Identity: nn.quantizable.Identity,
```

Comment:
Are we adding this here because of the additional Dequant stub? We shouldn't need this ideally. 
I would expect the model to go from  (assuming only Embeddings are being QAT'ed here)
fp32
`x -> nn.EmbeddingBag(x) -> nn.Linear() -> ... -> output`
to 
qat fp32 model 
`x -> nn.qat.EmbeddingBag(x) -> nn.Linear() -> ... -> output`